.github/workflows/nightly-test-integrations.yml

```diff
@@ -309,13 +309,107 @@ jobs:
         run: datadog-ci junit upload --service "$GITHUB_REPOSITORY" $TEST_RESULTS_DIR/results.xml
 
 
+  peering_commontopo-integration-test:
+    runs-on: ${{ fromJSON(needs.setup.outputs.compute-xl) }}
```

Comment:
@curtbushko I guess this should be `large` like the rest of them?

Reply:
It would be better if it was large but I am not 100% if it will even run on a large machine.  Maybe try?

Reply:
let's give it a try.